avocado/plugins/run.py

```diff
@@ -166,9 +166,10 @@ def run(self, args):
             sys.exit(exit_codes.AVOCADO_FAIL)
         job_instance = job.Job(args)
         job_run = job_instance.run()
-        result_dispatcher = ResultDispatcher()
-        if result_dispatcher.extensions:
-            result_dispatcher.map_method('render',
-                                         job_instance.result,
-                                         job_instance)
+        if job_instance.result.tests_total > 0:
+            result_dispatcher = ResultDispatcher()
+            if result_dispatcher.extensions:
+                result_dispatcher.map_method('render',
+                                             job_instance.result,
+                                             job_instance)
```

Comment:
As I mentioned, this was kept separately because I also do not believe in that idea, but would like to hear feedback.

Again, I have the feeling that a Job should stand on its own, having tests or not.  The same should be true for test result plugins, they should be prepared to deal with those situations.

So, the resolution for this commit is to drop it.